app/pb_kits/playbook/pb_textarea/_textarea.scss

```diff
@@ -2,9 +2,9 @@
 @import "./textarea_mixin";
 
 [class^=pb_textarea_kit] {
+  margin: 8px;
```

Comment:
Let's revert 😉

Reply:
Yep, I'll rebase onto Roxanne's branch for this